### ISSUE TITLE
Next release

### ISSUE THREAD

**Issue by pbrisbin** [mbj/stratosphere#223]
Hi there-

I was trying to use the `Tags` property on `LogGroup` and it looks like v0.60 doesn't know about it, but `main` does. Are you planning on a release any time soon?

**Comment by mbj** [comment]
@pbrisbin Yes, but I'm quite busy commercially. I mostly use the git references ATM.

There are 2 big things left before I could release:

1. Figure out on how to use the GHC AST to render comments, to bring docs back after the re-write of the generator into split packages and a different generation backend.
2. Script up the release of the new massive amount of packages.

As my current clients are okay we use the git references, and I have almost no time for OSS ATM: You can help in doing 1) or 2), alternatively you can also do 3) sponsor me (better make the org that relies on it sponsor me) on GH and this will give me more time I can spend towards these goals.

**Comment by mbj** [comment]
I've figured out module haddoc comments via the AST based generator. But the regular annotation fields cannot be nicely used to generate documentation without constructing the AST witht source locations, and this is something I explicitly want to avoid. 

I'm looking at other AST (and AST wrappers) to improve this.

**Comment by pbrisbin** [comment]
> drop support for pre 8.8

Won't affect us 👍 

> releasing this as lots of sublibraries

Is it possible to depend on a specific package's library/ies ? We use,

```yaml
      - services/applicationautoscaling
      - services/cloudfront
      - services/cloudwatch
      - services/ec2
      - services/ecs
      - services/elasticloadbalancingv2
      - services/events
      - services/iam
      - services/kms
      - services/lambda
      - services/logs
      - services/route53
      - services/s3
      - services/sns
      - services/stepfunctions
```

Which is far fewer than 283, so I think if this change meant downloading and compiling all the libraries, that'd be pretty bad. If there's a way to only download/compile specific libraries within a package, then it won't impact us.

Either way, you gotta do what you gotta do :)

**Comment by mbj** [comment]
@pbrisbin I've got no idea why docs are not being generated on hackage, if you have an idea what I can change let me know.

**Comment by pbrisbin** [comment]
```
Status
  Docs pending
  Build status unknown [[no reports yet](https://hackage.haskell.org/package/stratosphere-1.0.0/reports/)]
```

I _think_ this means it hasn't tried yet and we just have to wait. If it tried and failed, there'd be a report with errors there.

**Comment by mbj** [comment]
doc are present now. @pbrisbin could you switch of the git source? Releases are usable for you?

**Comment by pbrisbin** [comment]
I opened a PR to switch. It builds (with a minor change for `mkLoggingProperty`) and our tests pass. I suspect it'll get merged tomorrow (that team is backed up on code review), but can report back if anything goes awry.
